src/run/ci_provider/github_actions/provider.rs

```diff
@@ -23,7 +23,6 @@ pub struct GitHubActionsProvider {
     pub ref_: String,
     pub head_ref: Option<String>,
     pub base_ref: Option<String>,
-    pub commit_hash: String,
```

Comment:
It was abstracted away in a common implementation for all the providers in https://github.com/CodSpeedHQ/runner/pull/18.
This code should have been removed in this previous PR instead of this one.